lib/FusionInventory/Agent/Task/Inventory/Win32/Softwares.pm

```diff
@@ -411,4 +430,56 @@ sub _processMSIE {
 
 }
 
+# List of SQL Instances
+sub _getSqlInstancesList {
+    my (%params) = @_;
+    my $logger = $params{logger};
+	my $softwareName = $params{softwarename};
+	my $softwareVersion = $params{softwareversion};
+	
+	# Registry access for SQL Instances
+    my $machKey = $Registry->Open('LMachine', {Access => KEY_READ}) or $logger->error("Can't open HKEY_LOCAL_MACHINE key: $EXTENDED_OS_ERROR");
+    my $sqlinstancesList = $machKey->{"SOFTWARE/Microsoft/Microsoft SQL Server/Instance Names/SQL"};
+
+	# List of SQL Instances
+	foreach my $sqlinstance (keys %$sqlinstancesList) {
+		my $sqlinstanceName = $sqlinstance;
+		my $sqlinstanceValue = $sqlinstancesList->{$sqlinstance};
+		# Get version and edition for each instance
+		my ($sqlinstanceEditionValue,$sqlinstanceVersionValue) = _getSqlInstancesVersions(
+			%params,
+			SOFTVERSION	=> $softwareVersion,
+			NAME		=> $sqlinstanceName,
+			VALUE		=> $sqlinstanceValue
+		);
+
```

Comment:
You should insert here a `next unless $sqlinstanceEditionValue && $sqlinstanceVersionValue;`

Then you don't need to return faked values at line 481.